jax/_src/numpy/ufuncs.py

```diff
@@ -1616,8 +1657,9 @@ def power(x1: ArrayLike, x2: ArrayLike, /) -> Array:
   # Handle cases #2 and #3 under a jit:
   return _power(x1, x2)
 
-# Array API alias
-pow = power
+def pow(x1: ArrayLike, x2: ArrayLike, /) -> Array:
+  """Alias of :func:`jax.numpy.power`"""
```

Comment:
I think `pow` may have to be added to the `aliases` list in the tests

Reply:
Done